index.js

```diff
@@ -0,0 +1,4 @@
+let drop = document.querySelector(".Header--dropdown")
```

Comment:
The drop down is not working for me here. You may want to take another look at this!